server/sonar-server/src/main/java/org/sonar/server/issue/index/IssueIndexer.java

```diff
@@ -143,7 +144,7 @@ public void deleteByKeys(String projectUuid, List<String> issueKeys) {
     BulkRequestBuilder builder = esClient.prepareBulk();
     for (String issueKey : issueKeys) {
       builder.add(esClient.prepareDelete(INDEX_TYPE_ISSUE, issueKey)
-        .setRefresh(false)
+        .setRefresh(REFRESH_NONE) // ES 5: change to setRefreshPolicy
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ATrailingCommentCheck)